force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -94,7 +149,15 @@ public with sharing class VariablesDataTypesOperators {
      * @return true if the number is positive, false otherwise.
      */
     public static Boolean isPositive(Integer num) {
-        return null; // Replace null with the variable you used to store the result
+        boolean isPositive; 
+        if(num > 0){
+            isPositive = true;
+            return isPositive;
+        }
+        Else {
+            return isPositive = false;
```

Comment:
This technically works, but this is a bad habit to do and can result in some unexpected results. Better to just assign the variable and then return it below the if statement. Then you can have a single return statement.